dataactcore/utils/report.py

```diff
@@ -1,18 +1,26 @@
 import itertools
 from operator import attrgetter
 
-from dataactcore.models.lookups import FILE_TYPE
+from dataactcore.models.lookups import FILE_TYPE, FILE_TYPE_DICT_NAME_LETTER
 
 
 def report_file_name(submission_id, warning, file_type, cross_type=None):
     """Format the csv file name for the requested file.
     @todo: unify these file names"""
     if cross_type:
         report_type_str = 'warning_' if warning else ''
-        return "submission_{}_cross_{}{}_{}.csv".format(submission_id, report_type_str, file_type, cross_type)
+        file_letters = ''
+        if file_type in FILE_TYPE_DICT_NAME_LETTER and cross_type in FILE_TYPE_DICT_NAME_LETTER:
+            file_letters = 'File_{}_to_{}_'.format(FILE_TYPE_DICT_NAME_LETTER[file_type],
+                                                   FILE_TYPE_DICT_NAME_LETTER[cross_type])
+        return "submission{}_crossfile_{}{}{}_{}.csv".format(submission_id, report_type_str, file_letters, file_type,
+                                                             cross_type)
     else:
-        report_type_str = 'warning' if warning else 'error'
-        return "submission_{}_{}_{}_report.csv".format(submission_id, file_type, report_type_str)
+        report_type_str = 'warning_' if warning else 'error_'
+        file_letters = ''
+        if file_type in FILE_TYPE_DICT_NAME_LETTER:
+            file_letters = 'File_{}_'.format(FILE_TYPE_DICT_NAME_LETTER[file_type])
+        return "submission{}_{}{}_{}report.csv".format(submission_id, file_letters, file_type, report_type_str)
```

Comment:
We have also lost an underscore here after `submission`

Reply:
See other comment